nisystemlink/clients/dataframe/utilities/_pandas_dataframe_operations.py

```diff
@@ -0,0 +1,164 @@
+from typing import Optional
+
+import pandas as pd
+from nisystemlink.clients.dataframe import DataFrameClient
+from nisystemlink.clients.dataframe.models import (
+    AppendTableDataRequest,
+    CreateTableRequest,
+    DataFrame,
+    QueryDecimatedDataRequest,
+    QueryTableDataRequest,
+)
+
+from ._pandas_utils import (
+    _get_table_index_name,
+    _infer_dataframe_columns,
+    _infer_index_column,
+)
+
+
+def create_table_from_pandas_df(
+    client: DataFrameClient, df: pd.DataFrame, table_name: str, nullable_columns: bool
+) -> str:
+    """Create a table from a pandas DataFrame.
+
+    Args:
+        client (DataFrameClient): Instance of DataFrameClient.
+        df (pd.DataFrame): Pandas dataframe.
+        table_name (str): Name of the table.
+        nullable_columns (bool): Make the columns nullable. Nullable columns can contain `null` values.
```

Comment:
Can we have a default value to `nullable_columns` field? 

@santhoshramaraj What do you think?

Reply:
We need to understand the impact of `nullable_columns` on performance to choose the default value or require the user to make a conscious decision.